testgrid/config.yaml

```diff
@@ -4270,6 +4270,8 @@ dashboards:
     test_group_name: ci-kubernetes-node-kubelet-serial-cpu-manager
     alert_options:
       alert_mail_to_addresses: 'balaji.warft@gmail.com, connor.p.d@gmail.com'
+  - name: gci-gke-autoscaling-gpu
```

Comment:
Can you change the name to `gke-device-plugin-gpu-autoscaling` and move it after `gke-device-plugin-gpu-monitoring`.

Also, add
```
     base_options: 'exclude-filter-by-regex=^BeforeSuite$'
     alert_options:
       alert_mail_to_addresses: 'gke-kubernetes-accelerators-bugs@google.com'
```